acceptance-tests/simple-permissioning-smart-contract/contracts/SimpleAccountPermissioning.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity >=0.4.0 <0.6.0;
+// THIS CONTRACT IS FOR TESTING PURPOSES ONLY
+// DO NOT USE THIS CONTRACT IN PRODUCTION APPLICATIONS
+
+contract SimpleAccountPermissioning {
+    address[] private whitelist;
```

Comment:
While there's no need to return the whole whitelist I think it's easier to just make this `mapping(address -> boolean)` rather than do the array manipulation and looping

Reply:
done